MatrixSDKTests/MXCryptoTests.m

```diff
@@ -3142,11 +3141,9 @@ - (void)testEncryptionAlgorithmChange
                 XCTAssertEqual(liveTimeline.state.encryptionAlgorithm.length, 0);   // with a nil algorithm
                 XCTAssertTrue(roomFromAlicePOV.summary.isEncrypted);
                 
-                // -> It must be impossible to send a messages (because the algorithm is not supported)
+                // -> It is still possible to send a message because crypto will use backup algorithm (which can never be removed)
```

Comment:
Can I ask if this is the same new behavior on android and web?
I am happy with the change. I just want to check the consistency.

Reply:
As far as I understand the convo in the crypto room, this is fine if the SDK continues to encrypt messages using the previously defined algorithm. This is the app that should alarm the end user about the wrong encryption configuration (https://github.com/vector-im/element-ios/issues/5322).

I am now happy about this test change.